types/InflationData.md

```diff
@@ -55,7 +55,31 @@ The query id is the keccak256 hash of the query data:
 ```s
 keccak256(queryData)
 ```
+`0xe5aebfa18b69338df426c64bfab35c1168e20c94ea5a355c26599ae34bfb41f9`
+
+**The query data for the Beauro of Labor Statistics Shelter Category Unadjusted Index should be formed as follows:**
+
+*Query Descriptor:*
 
+```json
+{"type":"InflationData","location":"USA","agency":"BLS","category":"shelter","descriptor":"unadjustedIndex"}
+```
+
+*queryData:*
+
+```s
+abi.encode("InflationData", abi.encode("USA","BLS","shelter","unadjustedIndex"))
+```
+
+`00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000000d496e666c6174696f6e44617461000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000180000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000140000000000000000000000000000000000000000000000000000000000000000355534100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000003424c5300000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000007686f7573696e67000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000005696e646578000000000000000000000000000000000000000000000000000000`
+
+## Query ID
+
+The query id is the keccak256 hash of the query data:
+
+```s
+keccak256(queryData)
+```
 `0x675bc9f0a8e2819955d38f450f10f89a3c86e38b962bf03e57d39b25ea4550c3`
```

Comment:
and this queryId
`0x85c2b4b9a77f4985bab312d6eef5f08945070e1c84dc456a0eee4578bf6f64d8`